src/main/java/dk/kb/labsapi/ImageExport.java

```diff
@@ -147,9 +155,62 @@ public QueryResponse illustrationSolrCall(String query, Integer startTime, Integ
         solrQuery.setHighlight(false);
         solrQuery.set(GroupParams.GROUP, false);
 
+        QueryResponse response = callSolr(solrQuery);
+        return response;
+    }
+
+    public QueryResponse fullpageSolrCall(String query, Integer startTime, Integer endTime, Integer max) throws IOException {
+        validateQueryParams(startTime, endTime, max);
+        int usableStartTime = setUsableStartTime(startTime);
+        int usableEndTime = setUsableEndTime(endTime);
+
+        // Construct solr query with filter
+        String filter = "recordBase:doms_aviser AND py:[" + usableStartTime + " TO "+ usableEndTime + "]";
+        log.info("The query gets filtered with the following filter: " + filter);
+        SolrQuery solrQuery = new SolrQuery();
+        solrQuery.setQuery(query);
+        solrQuery.setFilterQueries(filter);
+        solrQuery.setFields("pageUUID, page_width, page_height");
+        solrQuery.setRows(max == -1 ? defaultExport : max);
+        solrQuery.setFacet(false);
+        solrQuery.setHighlight(false);
+        solrQuery.set(GroupParams.GROUP, false);
+
+        QueryResponse response = callSolr(solrQuery);
+        return response;
+    }
+
+    /**
+     * Validate query parameters that are to be used for image extraction.
+     * @param startTime of timespan for query.
+     * @param endTime of timespan for query.
+     * @param max number of queries.
+     */
+    private void validateQueryParams(Integer startTime, Integer endTime, Integer max) throws IOException {
+        // Check start and end times
+        int usableStartTime = setUsableStartTime(startTime);
+        int usableEndTime = setUsableEndTime(endTime);
+        log.info("Usable start time is: " + usableStartTime);
+        log.info("usable end time is: " + usableEndTime);
```

Comment:
Please log as single log entry to guard against parallel execution. And please lower to `debug` as this is not a very important message.